docs/docs/10-addons.md

```diff
@@ -22,4 +22,5 @@ The add-ons below are in the development (unminified) version of React only:
 
 To get the add-ons, use `react-with-addons.js` (and its minified counterpart) rather than the common `react.js`.
```

Comment:
ReactWithAddons.js for the browser isn't going anywhere yet so we should probably make it clear that `React.addons.{addon}` exists there.